### ISSUE TITLE
MKR WIFI 1010

### ISSUE THREAD

**Issue by grezco** [arduino-libraries/WiFi101OTA#10]
Hello,

Is it possible to use this library on a MKR WIFI 1010 ?    
With WiFiNINA Library instead of WiFi101 ?

Thanks for this great work!

**Comment by facchinm** [comment]
Hi @grezco ,
not yet; there are a couple of changes that should be made; first of all we need to add `UDP.beginMulti` API to WiFiNINA class; then the library should be made a bit more configurable to avoid any change in the library itself.
I'll keep you posted as soon as all the pieces are in place!

**Comment by sandeepmistry** [comment]
@facchinm  we have `UDP.beginMulticast(...)` in both WiFi101 and WiFiNINA. So this library can be changed to use that.

The only other awkward things would be this libraries name :)

**Comment by facchinm** [comment]
@sandeepmistry right, the library required `UDP.beginMulti` which looks a bit non standard :wink: 
@grezco I pushed a PR to allow you to test the OTA easily https://github.com/arduino-libraries/WiFi101OTA/pull/11 , let us know if it works as intended!

**Comment by grezco** [comment]
Hi @facchinm ,
I tried to test the change in the pull request #11 but I never see the ip port in Arduino 1.8.7.
Could you tell me how I can debug this ?

**Comment by grezco** [comment]
Hi,
It seems to me that my connection problem was more on the IDE than the library.
It works with the change in #11 

Thank you

**Comment by facchinm** [comment]
Great to hear that! Leaving the issue open to make sure we don't forget about this

**Comment by Ginodel** [comment]
Ohhh... i opened up the same request in at the WifiNINA library section ... 
Is soon as something to test is available i'll test ... 

Thx for the work ... Gino

**Comment by jurgen178** [comment]
`beginMulticast` is working in [#11](https://github.com/arduino-libraries/WiFi101OTA/pull/11).
Thx!

**Comment by Fan1117** [comment]
Hi,

I use WiFiUdp to send data on MKR WiFi 1010. However, the latency is too high (2ms - 4ms). Does anyone know how to make it lower? 

Thank you!
